packages/now-cli/src/util/dev/server.ts

```diff
@@ -1102,7 +1106,7 @@ export default class DevServer {
     }
 
     const method = req.method || 'GET';
-    this.output.log(`${chalk.bold(method)} ${req.url}`);
+    this.output.debug(`${chalk.bold(method)} ${req.url}`);
```

Comment:
We don't want to show the URL anymore?

Reply:
No I don't think so, because:

1. It could conflict / interfere with the frontend dev server's own logging
1. It ends up being very noisey for i.e. loading a Next.js page since there are a lot of assets that get loaded

Reply:
Ok good point!